include/RAJA/pattern/WorkGroup/WorkStorage.hpp

```diff
@@ -280,24 +285,25 @@ class WorkStorage<RAJA::array_of_pointers, ALLOCATOR_T, Dispatcher_T>
 
   using const_iterator = random_access_iterator<const_iterator_base>;
 
-
   explicit WorkStorage(allocator_type const& aloc)
-    : m_vec(0, aloc)
-    , m_aloc(aloc)
-  { }
+      : m_vec(0, aloc),
+        m_aloc(aloc)
```

Comment:
I guess my way of placing commas was not normal. Hopefully clang format understands that it can't pull commas out of ifdefs.